docs/new_task_guide.md

```diff
@@ -99,6 +89,36 @@ Now, in our YAML config file we'll use the `!function` constructor, and tell the
 process_docs: !function utils.process_docs
 ```
 
+### Using Local Datasets
+
+To load a local dataset for evaluation, you can specify data files in the `dataset_kwargs` field, such as the following for JSON files:
+
+```
+dataset_path: json
+dataset_name: null
+dataset_kwargs:
+  data_files: /path/to/my/json
+```
+Or with files already split into separate directories:
+
+```
+dataset_path: arrow
+dataset_kwargs:
+  data_files:
+    train: /path/to/arrow/train/data-00000-of-00001.arrow
+    validation: /path/to/arrow/validation/data-00000-of-00001.arrow
+```
+
+Alternatively, if you have previously downaloaded a dataset from huggingface hub and wish to use the local files, you can will need to use `data_dir` under `dataset_kwargs` to point to where the directory is.
```

Comment:
`can will` -> `will`
`downaloaded` -> `downloaded`

Mention that these latter two are compatible with HF datasets' `save_to_disk()` explicitly?

Reply:
Fixed